### PR TITLE
centered as per login button

### PR DESCRIPTION
**Pull request by iRahulP**
Centered as per login button, and text visuals changed to white.

Will try to update the final one once fixed for the mobile version.

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -54,7 +54,8 @@ nav .logo {
 nav .user-greet {
   display: none;
   font-size: 20px;
-  margin-top: 10px;
+  margin-top: 20px 30px 0 0;
```
Comment (harshith-venkatesh) [inline css/main.css:57]: Could you please look into this...
It should be `margin`

https://developer.mozilla.org/en-US/docs/Web/CSS/margin

File: css/main.css [hunk 2]
```diff
@@ -54,7 +54,8 @@ nav .logo {
 nav .user-greet {
   display: none;
```
Comment (harshith-venkatesh) [inline css/main.css:55]: Why do we have `display:none` here?

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

File: css/main.css (hunk 2 above)
Comment (iRahulP) [reply css/main.css:55]: It was already there initially from the initial author for the code base, Should I update or just remove it.

**Comment by iRahulP** [comment]
snap

Current preview after update, also removed the display:none as it was already overridden by the root property.

**Comment by Vividh25** [comment]
> 
> 
> snap
> 
> Current preview after update, also removed the display:none as it was already overridden by the root property.

Can you post a screenshot for mobile version ?

**Comment by iRahulP** [comment]
mobile

Current layout for mobile view.

**Comment by Vividh25** [comment]
> 
> 
> mobile
> 
> Current layout for mobile v

File: css/main.css [hunk 3]
```diff
@@ -52,9 +52,9 @@ nav .logo {
 }
 
 nav .user-greet {
-  display: none;
   font-size: 20px;
-  margin-top: 10px;
+  margin: 20px 30px 0 0 !important;
```
Comment (harshith-venkatesh) [inline css/main.css:56]: Sorry for the late response, could we look into the way , where we don't use `!important` please

**Review by harshith-venkatesh — CHANGES_REQUESTED** [review]

**Comment by rohan09-raj** [comment]
This has been taken care of in PR #178 as there were new design changes in mobile view.

**Pull request closed without merging**